Source/Fuse.Controls.WebView/WebView.ScriptClass.uno

```diff
@@ -47,6 +51,47 @@ namespace Fuse.Controls
 		}
 
 		/**
+			Go back to the previous page.
+
+			@scriptmethod goBack()
+		*/
+		static void goBack(Context c, WebView view, object[] args)
+		{
+			view.GoBack();
+		}
+
+		/**
+			Go forward to the next page.
+
+			@scriptmethod goForward()
+		*/
+		static void goForward(Context c, WebView view, object[] args)
+		{
+			view.GoForward();
+		}
+
+		/**
+			Reload the current page.
+
+			@scriptmethod reload()
+		*/
+		static void reload(Context c, WebView view, object[] args)
+		{
+			view.Reload();
+		}
+
+
+		/**
+			Stop loading the page.
+
+			@scriptmethod stop()
+		*/
+		static void stop(Context c, WebView view, object[] args)
+		{
+			view.Stop();
+		}
```

Comment:
It seems to me like the other script-methods verify their arguments and raise a UserError if they're not the expected lenght. Any reason why these methods don't?